public/app/features/templating/template_srv.ts

```diff
@@ -129,8 +129,8 @@ export class TemplateSrv implements BaseTemplateSrv {
       format = 'glob';
     }
 
-    // some formats have arguments that come after ':' character
-    let args = format.split(':');
+    // some formats have arguments that come after ':' character but allow escaping of the : with a backslash for regex
+    let args = format.split(/(?<!\\):/);
```

Comment:
hmm, is this breaking backward compability?

Reply:
I don't _think_ it is, but I'm not 100% sure. I didn't see any cases in the previously existing format functions where preceding a colon with a backslash seemed likely or like a valid value. There are definitely no test cases covering it. I'm happy to consider other suggestions though.

The main reason for doing this is to support colons within the regex and/or the replacement. Without the escaping, the split function would treat them as additional args.